it/src/main/java/org/corfudb/universe/node/server/vm/VmCorfuServerParams.java

```diff
@@ -36,4 +37,22 @@ public VmCorfuServerParams(
         );
         this.vmName = vmName;
     }
+
+    @Builder
+    @EqualsAndHashCode
+    @Getter
+    public static class VmName implements Comparable<VmName> {
+        @NonNull
+        private final String host;
```

Comment:
Is this the host name?

Reply:
I think in vpshere the vm can have a different name than the host name.

Reply:
it is a vm name

Reply:
renamed host -> name